bsc-plugin/src/lib/rooibos/MockUtil.ts

```diff
@@ -186,4 +226,41 @@ export class MockUtil {
         }
     }
 
+    public addRuntimeGlobalFunctionMocks(file: BrsFile, astEditor: Editor) {
```

Comment:
Will be removing this function shortly as the logic has been consolidated with the `expectCalled` logic.

Reply:
done